packages/note/src/react/index.js

```diff
@@ -0,0 +1,214 @@
+import * as glamor from 'glamor'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Avatar from '@pluralsight/ps-design-system-avatar/react'
+import Icon from '@pluralsight/ps-design-system-icon/react'
+import { useTheme } from '@pluralsight/ps-design-system-theme/react'
+import {
+  arrayOfMaxLength,
+  elementOfType
+} from '@pluralsight/ps-design-system-prop-types'
+import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
+
+import css from '../css/index.js'
+
+const styles = {
+  note: (themeName, props) =>
+    glamor.compose(
+      glamor.css(css['.psds-note']),
+      glamor.css(css[`.psds-note.psds-theme--${themeName}`])
+    ),
+
```

Comment:
Kind of an interesting order, with note up top separated by a newline.  It leaves me curious as to the meaning of the ordering... 🤔

Reply:
i don't think there was a reasoning behind it. just an oversite.